doc/releases/release-notes-3.1.rst

```diff
@@ -74,6 +74,10 @@ New APIs in this release
   * Added :c:macro:`IN_RANGE` for checking if a value is in the range of two
     other values.
 
+* MIPI
+
+  * Added a new MIPI API for MIPI-DSI drivers.
```

Comment:
mention that it is experimental (as it has some features/API calls that will likely be implemented later)